src/igvfd/schemas/matrix_file.json

```diff
@@ -102,12 +102,12 @@
             ],
             "enum_descriptions": {
                 "contact matrix": "A matrix used in genomic assays, particularly in chromosome conformation capture techniques like Hi-C, to represent the frequency of physical interactions between different regions of the genome.",
+                "filtered feature barcode matrix": "A matrix output from Space Ranger, where rows represent features and columns represent barcoded spots or cells. It contains counts of sequencing reads after filtering out low-quality or low-abundance features.",
+                "raw feature barcode matrix": "A matrix output from Space Ranger, where rows represent features and columns represent barcoded spots or cells. It contains raw counts of sequencing reads before any filtering or normalization.",
```

Comment:
While you are in here can you change this from "Space Ranger" to "Cell Ranger", please?